addons/website/static/src/components/editor/editor.js

```diff
@@ -82,7 +81,10 @@ export class WebsiteEditorComponent extends Component {
     }
 
     destroyAfterTransition() {
-        this.websiteContext.edition = false;
+        this.state.showWysiwyg = false;
+        this.props.reloadIframe().then(() => {
```

Comment:
It looks like a good idea, to reload after removing the wysiwyg! Maybe we could also use the block iframe component while reloading (I know I asked to put as less loading screen as possible at first, but it looks useful now, sorry for that :sweat_smile:). Also, is there a need for .then instead of await ?